tests/test_e2e_10_mef_eline.py

```diff
@@ -613,3 +693,649 @@ def test_080_create_and_remove_ten_circuits_ten_times(self):
+        }
+
+        # It sets a new circuit's start_date
+        response = requests.patch(api_url + evc1, data=json.dumps(payload),
+                                  headers={'Content-type': 'application/json'})
+        assert response.status_code == 200
+
+        time.sleep(10)
+
+        # It verifies EVC's data
+        response = requests.get(api_url + evc1)
+        data = response.json()
+        assert data['start_date'] == start.strftime(TIME_FMT)
+
+    def test_110_patch_end_date(self):
+
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        evc1 = self.create_evc(100)
+
+        end_delay = 60
+        end_date = datetime.now() + timedelta(minutes=end_delay)
+
+        payload = {
+            "end_date": end_date.strftime(TIME_FMT)
+        }
+
+        # It sets a new circuit's end_date
+        requests.patch(api_url + evc1, data=json.dumps(payload),
+                       headers={'Content-type': 'application/json'})
+        time.sleep(10)
+
+        # It verifies EVC's data
+        response = requests.get(api_url + evc1)
+        data = response.json()
+        assert data['end_date'] == end_date.strftime(TIME_FMT)
+
+    def test_115_patch_bandwidth(self):
+
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        evc1 = self.create_evc(100)
+
+        bandwidth = 40
+        payload = {
+            "bandwidth": bandwidth
+        }
+
+        # It sets a new circuit's bandwidth
+        requests.patch(api_url + evc1, data=json.dumps(payload),
+                       headers={'Content-type': 'application/json'})
+        time.sleep(10)
+
+        # It verifies EVC's data
+        response = requests.get(api_url + evc1)
+        data = response.json()
+        assert data['bandwidth'] == bandwidth
+
+        requests.delete(api_url + evc1)
+
+    def test_120_patch_priority(self):
+
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        evc1 = self.create_evc(100)
+
+        priority = 100
+        payload = {
+            "priority": priority
+        }
+
+        # It sets a new circuit's priority
+        requests.patch(api_url + evc1, data=json.dumps(payload),
+                       headers={'Content-type': 'application/json'})
+
+        time.sleep(10)
+
+        # It verifies EVC's data
+        response = requests.get(api_url + evc1)
+        data = response.json()
+        assert data['priority'] == priority
+
+    def test_125_patch_queue_id(self):
+
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        evc1 = self.create_evc(100)
+
+        queue_id = 100
+        payload = {
+            "queue_id": queue_id
+        }
+
+        # It sets a new circuit's queue_id
+        requests.patch(api_url + evc1, data=json.dumps(payload),
+                       headers={'Content-type': 'application/json'})
+
+        time.sleep(10)
+
+        # It verifies EVC's data
+        response = requests.get(api_url + evc1)
+        data = response.json()
+        assert data['queue_id'] == queue_id
```

Comment:
Here we can check the flows using dump-flows as well

Reply:
This is resolved with commit [b735c67](https://github.com/amlight/kytos-end-to-end-tests/pull/69/commits/b735c672add6f4300632ad241bc55e3d7853cf0c)